src/doc/en/developer/workflows.rst

```diff
@@ -2,200 +2,176 @@
 
 .. _chapter-workflows:
 
-=======================
-Distributed Development
-=======================
+=====================
+Using Git with GItHub
```

Comment:
lowercase i

Reply:
Done. Thanks. 

Note that I also decapitalized all subsection titles.